lerna.json

```diff
@@ -2,7 +2,7 @@
   "packages": [
     "projects/*"
   ],
-  "version": "13.0.0",
+  "version": "14.0.0",
```

Comment:
In the future, don't version in the PR. The release tooling will automate versioning for me.

Reply:
Thanks! Please let me know if I need to revert the version changes.

Reply:
Thanks. The version changes are reverted.